core/src/org/pentaho/di/core/exception/IdNotFoundException.java

```diff
@@ -0,0 +1,103 @@
+/*! ******************************************************************************
```

Comment:
Please add unit tests for this class (it will be very simple but it will help code coverage). Be sure the check/specify when null should and should not be valid.